src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -15,10 +15,10 @@
 
 from grpc import _common
 from grpc._cython import cygrpc
-from grpc.experimental import aio
 
 
-class UnaryUnaryMultiCallable(aio.UnaryUnaryMultiCallable):
+class UnaryUnaryMultiCallable:
+    """Affords invoking a unary-unary RPC from client-side in an asynchronous way."""
```

Comment:
s/a/an

Reply:
OK

Reply:
Nit: Actually, `s/an/a/`. While, `u` the grapheme is a generally a vowel, the word [is pronounced `/ˈjuːnəɹi/`](https://en.wiktionary.org/wiki/unary), so the first *phoneme*, `j`, is a consonant. Hence "a".

Reply:
Hi Richard, thanks.
`Afford invoking a unary-unary RPC from client-side in an asynchronous way. `
Is it right?